livre/Eq/Config.cpp

```diff
@@ -67,10 +74,9 @@ class Config
         , volumeBBox( Boxf::makeUnitBox( ))
 #ifdef LIVRE_USE_ZEQ
         , _publisher( lunchbox::URI( "hbp://" ))
-        , _rcPublisher( lunchbox::URI( "restresp://" ))
+        , _rcPublisherPtr( 0 )
```

Comment:
Sure ?? I think it should be _restPublisherPtr because _publisher is not a ptr. We should be able to differentiate.